web.py

```diff
@@ -213,6 +214,27 @@ def test_ws():
             flash('Event not yet implemented', 'danger')
     return render_template('test_ws.html', recipient=recipient, event=event, events=events)
 
+@roles_accepted(Role.ROLE_ADMIN, Role.ROLE_FINANCE)
+@app.route('/user_compliance', methods=['GET', 'POST'])
+def user_compliance():
+    email = ''
+    if request.method == 'POST':
+        email = request.form['email']
+        user = User.from_email(db.session, email)
+        if user:
+            kycrequest = KycRequest.from_user_id(db.session, user.id)
+            if kycrequest:
+                token = kycrequest.token
+                pdf = kyc_core.download_pdf_backup(token)
+                if not pdf:
```

Comment:
nicer error using `flash()`
```suggestion
                if pdf:
                    response = make_response(pdf)
                    response.headers.set('Content-Type', 'application/pdf')
                    return response
                else:
                   flash('failed to download pdf', 'danger')
```

Reply:
committed the change above